lisa/src/main/java/it/unive/lisa/util/datastructures/graph/FixpointGraph.java

```diff
@@ -220,6 +221,9 @@ protected FixpointGraph(G other) {
 				try {
 					newIntermediate = (F) mkInternalStore(entrystate);
 					newApprox = semantics.compute(current, entrystate, cg, newIntermediate);
+					// FIXME the state should be cleaned through push/pop, not
+					// through a cleanup here!
+					// newApprox = cleanUpPostState(current, newApprox);
```

Comment:
This cleanup just removes the variables that go out of scope. The push/pop having a call as "token" do not fit here